### DISCUSSION TITLE
Allow basic formatting in notes

### DISCUSSION THREAD

**Discussion by webster8347** [karakeep-app/karakeep#498] [Ideas]
Cool stuff! Please allow basic formatting in notes, e.g. Bold, Italics, Tabs, Lists... actually what you see here: 

![Screenshot 2024-10-06 181758](https://github.com/user-attachments/assets/4f221e5d-862b-4cf0-81cf-7a9c904f0325)

**Comment by kamtschatka** [comment]
already works, you just have to use markdown (see https://www.markdownguide.org/basic-syntax/)

**Comment by webster8347** [comment]
Oh, right, sorry :-)
